draft-data/main.py

```diff
@@ -0,0 +1,7 @@
+
+
+def read_text(file_name):
```

Comment:
this function should be changed to enhance performance

Reply:
the issue has been addressed